lel.py

```diff
@@ -0,0 +1,11 @@
+import traceback
```

Comment:
Really? 🧐

Reply:
Ah, you discovered the secret file. You won 100000$ :)